diesel_derives/src/util.rs

```diff
@@ -95,6 +95,7 @@ where
 pub fn wrap_in_dummy_mod(item: TokenStream) -> TokenStream {
     quote! {
         #[allow(unused_imports)]
+        #[allow(unused_qualifications)] // Can remove if https://github.com/rust-lang/rust/issues/130277 gets done
```

Comment:
it's probably best to not put the comment in `quote!`, because I would expect that to get the comment to be expanded as well, but we probably really only want the comment within Diesel, not in the expanded code.

Reply:
[8b8c43e](https://github.com/diesel-rs/diesel/pull/4259/commits/8b8c43ee9471bd85f9dbaf9c1e4dc4dd67c59493)

Reply:
[5c95a6b](https://github.com/diesel-rs/diesel/pull/4259/commits/5c95a6bd510d1194746cd0c89278d1af65dd5653)